app/assets/javascripts/discourse/app/mixins/composer-video-thumbnail-uppy.js

```diff
@@ -103,6 +103,7 @@ export default Mixin.create(ExtendableUploader, UppyS3Multipart, {
             });
           }
         });
+        callback();
```

Comment:
Should the callback be called here or at the and of the function passed to `canvas.toBlob()` above? (i.e. it might be called too early now)

Reply:
Oh, I see what you mean. I did have two timeouts originally, and so I think I left this callback there as part of that setup. Let me double check...

Reply:
Ah, turns out the callback should be in the on upload-success trigger